client/src/components/Settings.js

```diff
@@ -5,32 +5,67 @@ function Settings(props) {
   const {color, size} = props.state
 
   const increaseSpeed = () => {
-    const lightElem = document.getElementById("elem");
+    let lightElem = document.getElementById("elem");
     if (lightSpeed > 1) {
       lightSpeed -= 1;
     }
     lightElem.style = 'animation: ani ' + lightSpeed + 's infinite';
   };
 
-
   const decreaseSpeed = () => {
-    const lightElem = document.getElementById("elem");
+    let lightElem = document.getElementById("elem");
     lightSpeed += 1;
     lightElem.style = 'animation: ani ' + lightSpeed + 's infinite';
   }
 
+  const changeShape = (className) => {
+    let lightElem = document.getElementById("elem");
+    let elemStyle = lightElem.style.cssText.split(';');
+
+    if (className.indexOf('square') > -1) {
+      elemStyle.push('border-radius: 0');
```

Comment:
do we have to change the css directly like this? 

when the `circle` class is added to the element, can it just get the appropriate css properties from the class itself?

Reply:
For sure. I did this as a quick check to handle changing the styles to see the updates instead of investing in putting together a helper function for class handling. Will be doing the class handling updates on the next commits.